cmd/podman/common/volumes.go

```diff
@@ -156,25 +178,13 @@ func getMounts(mountFlag []string) (map[string]spec.Mount, map[string]*specgen.N
 	finalMounts := make(map[string]spec.Mount)
 	finalNamedVolumes := make(map[string]*specgen.NamedVolume)
 
-	errInvalidSyntax := errors.Errorf("incorrect mount format: should be --mount type=<bind|tmpfs|volume>,[src=<host-dir|volume-name>,]target=<ctr-dir>[,options]")
-
-	// TODO(vrothberg): the manual parsing can be replaced with a regular expression
-	//                  to allow a more robust parsing of the mount format and to give
-	//                  precise errors regarding supported format versus supported options.
 	for _, mount := range mountFlag {
-		arr := strings.SplitN(mount, ",", 2)
-		if len(arr) < 2 {
-			return nil, nil, errors.Wrapf(errInvalidSyntax, "%q", mount)
+		// TODO: Docker defaults to "volume" if no mount type is specified.
```

Comment:
Why not just make this default in findMountType?

Reply:
See commit message:

> Note that this is the ground work to default to "volume" (again Docker
compat). However, this will require some further massaging as we have
to assign a name.

Defaulting to "volume" fails as volumes need to be named. I am sure we can tackle it much easier now but refrained from doing that in the bug fix.

Reply:
Volumes don't technically need to be named. If you pass libpod a named volume with no name, we will make an anonymous volume (pseudorandom ID as name)

Reply:
Curious. I got an error when playing with the default. Maybe it was a red herring.